src/base64.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 export function base64(i) {
-  return (new Buffer(i, 'ascii')).toString('base64');
+  return (new Buffer.from(i, 'ascii')).toString('base64');
```

Comment:
Just curious, why is `Buffer.from` preferred?

Reply:
> Just curious, why is `Buffer.from` preferred?

The old way of instantiating `Buffer` via the constructor (Node 6 and before) yielded very different behavior based on the argument types.  A few new methods (including `Buffer.from`) came to be in a more recent node version to help disambiguate things and make it N methods vs one method that accepts N argument types.

https://nodejs.org/api/buffer.html#buffer_buffer_from_buffer_alloc_and_buffer_allocunsafe has a decent short write-up on how things work now.